views/settings/page-sections/dashboard.php

```diff
@@ -24,75 +24,90 @@
+			<img src="<?php echo esc_url( WP_ROCKET_ASSETS_IMG_URL ); ?>logo-adblock.svg" width="52" height="52" alt="Logo Adblock">
 			<div>
-				<div class="wpr-adblock-title"><?php _e( 'WP Rocket : Ad blocker detected.', 'rocket' ); ?></div>
-				<h2 class="wpr-adblock-description"><?php _e( 'Disable it on your site to access our support features. <a href="https://docs.wp-rocket.me/article/1080-disable-ad-blocker-for-support-integration" target="_blank">Learn more</a>', 'rocket' ); ?></h2>
+				<div class="wpr-adblock-title"><?php esc_html_e( 'WP Rocket : Ad blocker detected.', 'rocket' ); ?></div>
+				<h2 class="wpr-adblock-description"><?php esc_html_e( 'Disable it on your site to access our support features. <a href="https://docs.wp-rocket.me/article/1080-disable-ad-blocker-for-support-integration" target="_blank">Learn more</a>', 'rocket' ); ?></h2>
 			</div>
-			<a class="wpr-adblock-close wpr-icon-close rocket-dismiss" href="<?php echo wp_nonce_url( admin_url( 'admin-post.php?action=rocket_ignore&box=rocket_adblock_notice' ), 'rocket_ignore_rocket_adblock_notice' ); ?>"><span class="screen-reader-text"><?php _e( 'Dismiss this notice.', 'rocket' ); ?></span></a>
+			<a class="wpr-adblock-close wpr-icon-close rocket-dismiss" href="<?php echo esc_url( wp_nonce_url( admin_url( 'admin-post.php?action=rocket_ignore&box=rocket_adblock_notice' ), 'rocket_ignore_rocket_adblock_notice' ) ); ?>"><span class="screen-reader-text"><?php esc_html_e( 'Dismiss this notice.', 'rocket' ); ?></span></a>
 		</div>
 	</div>
-	<?php
+		<?php
 	endif;
 
-	if ( ! in_array( 'rocket_activation_notice', (array) $boxes, true ) ) : ?>
+	if ( ! in_array( 'rocket_activation_notice', (array) $rocket_boxes, true ) ) :
+		?>
 	<div class="wpr-notice">
 		<div class="wpr-notice-container">
-			<div class="wpr-notice-supTitle"><?php _e( 'Congratulations!', 'rocket' ); ?></div>
-			<h2 class="wpr-notice-title"><?php _e( 'WP Rocket is now activated and already working for you.<br>
-				Your website should be loading faster now!', 'rocket' ); ?></h2>
-				<div class="wpr-notice-description"><?php _e( 'To guarantee fast websites, WP Rocket applies 80% of web performance best practices.<br> We also enable options that provide immediate benefits to your website.', 'rocket' ); ?></div>
-				<div class="wpr-notice-continue"><?php _e( 'Continue to the options to further optimize your site!', 'rocket' ); ?></div>
-				<a class="wpr-notice-close wpr-icon-close rocket-dismiss" href="<?php echo wp_nonce_url( admin_url( 'admin-post.php?action=rocket_ignore&box=rocket_activation_notice' ), 'rocket_ignore_rocket_activation_notice' ); ?>"><span class="screen-reader-text"><?php _e( 'Dismiss this notice.', 'rocket' ); ?></span></a>
+			<div class="wpr-notice-supTitle"><?php esc_html_e( 'Congratulations!', 'rocket' ); ?></div>
+			<h2 class="wpr-notice-title">
+			<?php
+				esc_html_e( 'WP Rocket is now activated and already working for you.<br>
```

Comment:
We should write it without line break (like in the next message, few lines below). Like:
```... working for you.<br> Your website ...```